packages/components/statistic/src/countdown.ts

```diff
@@ -1,29 +1,43 @@
-import { divide, floor, multiply, padStart, reduce } from 'lodash-unified'
-import dayjs from 'dayjs'
+import { padStart } from 'lodash-unified'
 import { buildProps, definePropType } from '@element-plus/utils'
 import type { StyleValue } from 'vue'
 import type { Dayjs } from 'dayjs'
 import type Countdown from './countdown.vue'
 export const countdownProps = buildProps({
+  /**
+   * @description Formatting the countdown display
+   */
   format: {
     type: String,
     default: 'HH:mm:ss',
   },
+  /**
+   * @description Sets the prefix of a countdown
+   */
   prefix: {
     type: String,
-    default: '',
   },
+  /**
+   * @description Sets the suffix of a countdown
+   */
   suffix: {
     type: String,
-    default: null,
   },
+  /**
+   * @description countdown titles
+   */
   title: {
-    type: [String, Number],
-    default: '',
+    type: String,
   },
+  /**
+   * @description Styles countdown values
+   */
   value: {
     type: definePropType<number | Dayjs>([Number, Object]),
   },
+  /**
+   * @description numerical precision
```

Comment:
Also here.

In addition, I suggest that the type here be modified to:

```diff
valueStyle: {
-  type: definePropType<StyleValue>([String, Object]),
-  default: () => ({}),
+  type: definePropType<StyleValue>([String, Object, Array]),
}
```

Reply:
ok